filament/src/RenderPass.cpp

```diff
@@ -348,7 +354,9 @@ void RenderPass::generateCommandsImpl(uint32_t extraFlags,
 
         cmdColor.key = makeField(soaVisibility[i].priority, PRIORITY_MASK, PRIORITY_SHIFT);
         cmdColor.primitive.index = (uint16_t)i;
-        variant.setShadowReceiver(soaVisibility[i].receiveShadows & hasShadowing);
+        if (!Variant::isSSRVariant(variant)) {
```

Comment:
Same thing here? Is the compiler doing the right thing?

Reply:
okay, made it more branchless.